vivisect/parsers/elf.py

```diff
@@ -379,6 +394,36 @@ def loadElfIntoWorkspace(vw, elf, filename=None, baseaddr=None):
         elif sname == ".dynsym":
             [s for s in makeSymbolTable(vw, sva, sva+size)]
 
+        elif sname in (".bss",):
+            if vw.getName(fname + '.bss_temp') is None:
+                sdasz = sec.sh_size
+                align = sec.sh_addralign
+                sdasz += align-1
+                sdasz = (sdasz // align) * align
+                sdabase = vw.addMemoryMap(None, 7, fname, b'\0' * sdasz)
+                vw.addSegment(sdabase, sdasz, '.bss_temp', fname)
+                vw.makeName(sdabase, fname + ".bss_temp")
+
+        elif sname in (".sbss", ".sdata"):
+            if vw.getName('_SDA_BASE_') is None:
+                sdasz = sec.sh_size
+                align = sec.sh_addralign
+                sdasz += align-1
+                sdasz = (sdasz // align) * align
+                sdabase = vw.addMemoryMap(None, 7, fname, b'\0' * sdasz)
+                vw.addSegment(sdabase, sdasz, '.sda_base', fname)
+                vw.makeName(sdabase, "_SDA_BASE_")
+
+        elif sname in (".sbss2", ".sdata2"):
+            if vw.getName('_SDA2_BASE_') is None:
+                sdasz = sec.sh_size
+                align = sec.sh_addralign
+                sda2sz += align-1
+                sda2sz = (sda2sz // align) * align
+                sda2base = vw.addMemoryMap(None, 7, fname, b'\0' * sda2sz)
+                vw.addSegment(sdabase, sdasz, '.sda2_base', fname)
+                vw.makeName(sda2base, "_SDA2_BASE_")
+
```

Comment:
is all this still necessary?  or did we fix it all with the ELF-map-alignment fixes?